boards/arduino/nano_matter/arduino_nano_matter-pinctrl.dtsi

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2025 Pete Johanson
+ * Copyright (c) 2025 Silicon Laboratories Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/silabs/xg24-pinctrl.h>
+
+&pinctrl {
+	eusart0_default: eusart0_default {
+		group0 {
+			pins = <EUSART1_TX_PA4>;
+			drive-push-pull;
+			output-high;
+		};
+
+		group1 {
+			pins = <EUSART1_RX_PA5>;
```

Comment:
you prob want `EUSART0_TX_xx` here?

Reply:
Had to do a double take, since RX is correct in group1, but yes, agree it should be EUSART**0** both on the TX and RX side.

Reply:
ya the comment is one lines 13-19 actually, but depending on where you are Github might make things more confusing than they need to be :)